src/components/EndGame.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Button from './Button';
+import flame from '../assets/img/flame2.png';
+import BulletHole from './BulletHole';
+
+class EndGame extends Component {
+    static propTypes = {
+        winner: PropTypes.string.isRequired,
+        onTryAgain: PropTypes.func.isRequired
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.onEnter = this.onEnter.bind(this);
+
+        this.state = {
+            winner: '',
+            isTextVisible: false,
+            isNameVisible: false
+        }
+    }
+
+    componentDidMount() {
+        setTimeout(() => {
+            this.setState({
+                isTextVisible: true
+            })
+        }, 100);
+        setTimeout(() => {
+            this.setState({
+                isNameVisible: true
+            })
+        }, 1000);
+        document.addEventListener('keyup', this.onEnter);
```

Comment:
I would put this at the top of the function, and add empty lines, after it `setTimeout`